### DISCUSSION TITLE
Hamdashboard Downunder

### DISCUSSION THREAD

**Discussion by vk3ymml** [VA3HDL/hamdashboard#43] [General]
Hi VA3HDL

We love your version of dashboard.  We have customized ours more weather and radio sources for Australia. I am based in Melbourne, Victoria and I am studying to become and Amateur Radio Operator soon.

I also host and manage a Radio Scanner forum that is dedicated to the state of Victoria, Australia so our customization of Hamdashboard has been centred around this.

Our version is hosted at: https://www.vicscan.com/hamdash/

We are slowly working to incorporate further resources that are specifically Australian related

Thank you again
"VK3YMML" - (My Location *wink* not my call)
Melbourne, Australia

**Comment by VA3HDL** [comment]
Looks great!
Will add to the list of public installs.
Happy Holidays!

**Reply by vk3ymml** [reply]
Hi 
Can you please update my link for Hamdashboard as I now have a Foundation Amateur licence 

VK3VSN - Australia

**Reply by VA3HDL** [reply]
Hi Brett,

Page udpated.

73

**Comment by KJ5AJB** [comment]
Okay will take care of it. Sent from my iPhoneOn Sep 16, 2025, at 5:56 AM, Brett Robson ***@***.***> wrote:﻿
Hi
Can you please update my link for Hamdashboard as I now have a Foundation Amateur licence
VK3VSN - Australia
